loops-and-arrays.js

```diff
@@ -21,13 +25,17 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+for (let i = 0; i < numbers.length; i++) {
+  numbers[i] = numbers[i] * 3;
+  console.log(numbers[i]);
+}
+
```

Comment:
Prima stukje code wederom, alleen hetzelfde als bij de vorige opdracht. Zou je om het exacte antwoord te krijgen dat op 1 lijn staat. de console.log buiten de for loop kunnen plaatsen en enkel (numbers) aanroepen ipv (numbers[i]).